Nos/Views/SearchTextFieldObserver.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  SearchTextFieldObserver.swift
+//  Nos
+//
+//  Created by Martin Dutra on 3/4/23.
+//
+
+import Combine
+import Foundation
+import SwiftUI
+
+/// Class suitable for being used in a searchable modifier as it debounces
+/// the input to debouncedText.
+class SearchTextFieldObserver: ObservableObject {
+    @Published
+    var debouncedText = ""
+
+    @Published
+    var text = ""
+
+    private var subscriptions = Set<AnyCancellable>()
+
+    init() {
+        $text
+            .removeDuplicates()
+            .debounce(for: .milliseconds(500), scheduler: RunLoop.main)
```

Comment:
Let's go for 200 milliseconds like we do on the Discover tab

Reply:
👌